templates/SilverStripe/DiscovererSearchUI/Page/Layout/SearchResults.ss

```diff
@@ -3,5 +3,13 @@
         $SearchForm
     </div>
 
+    <% if $QuerySuggestions %>
+        <div class="discoverer-suggestions">
+            <h2>Search suggestions:</h2>
```

Comment:
does this need to be translatable?

Reply:
Yup, good shout.

I think there are some templates that are difficult to add translations to (like pagination overview, where you have a mix of strings and data), but any of this simple stuff I can definitely chuck `_t` in.